tests/compiler/literals.json

```diff
@@ -1,4 +1,10 @@
 {
   "asc_flags": [
+  ],
+  "stderr": [
+    "AS236: Literal '-9223372036854775809' does not fit into 'i64' or 'u64' types.",
+    "AS236: Literal '-0x8000000000000001' does not fit into 'i64' or 'u64' types.",
+    "AS236: Literal '-0o1000000000000000000001' does not fit into 'i64' or 'u64' types.",
+    "EOF"
```

Comment:
Looks like this will prevent generation of a fixture now, since compilation errors. The error cases should likely live in a separate `literals-errors` test.

Reply:
Ah, okey